waspc/data/Generator/templates/server/nodemon.json

```diff
@@ -4,7 +4,9 @@
   },
   "watch": [
     "src/",
+    "../../../src/",
     ".env"
   ],
+  "comment-filip": "We now have to watch ../../../src/ because we're importing client files directly",
```

Comment:
Todo(filip): Check if this is still necessary now that Martin implemented the `Watch` changes.